src/log.tf

```diff
@@ -24,10 +24,11 @@ resource "azurerm_storage_account" "diag-log" {
     type = "SystemAssigned"
   }
 
-  network_rules {
-    default_action = "Deny"
-    bypass         = ["AzureServices", "Logging"]
-  }
+  # network_rules {
```

Comment:
We want to keep these

Reply:
In the description of the PR I explained why I removed them